### PR TITLE
update to use new logo and font

### PR DESCRIPTION
**Pull request by drkane**

### CODE REVIEW

**Comment by BibianaC** [comment]
@drkane `LICENCE.rst` also needs to be updated with the new logo path.

**Comment by BibianaC** [comment]
[GrantNav rebrand - New Logo & Font](https://trello.com/c/CI6QqGZM/50-grantnav-rebrand-new-logo-font)

**Comment by drkane** [comment]
> 
> 
> @drkane `LICENCE.rst` also needs to be updated with the new logo path.

Good point - now updated. I've also deleted the old logos.

File: LICENCE.rst [hunk 1]
```diff
@@ -1,7 +1,9 @@
 GrantNav
 ========
 
-Note: This license doesn't apply to the 360Giving logo https://github.com/OpenDataServices/grantnav/blob/master/grantnav/frontend/static/images/360logo.png https://github.com/OpenDataServices/grantnav/blob/master/grantnav/frontend/static/images/360logo_invert.png . The 360Giving logo is copyright `360Giving <https://threesixtygiving.org>` and may not be used without express permission. 
+Note: This license doesn't apply to the 360Giving logo /grantnav/frontend/static/images/360-giving-logo-white.svg
+
+The 360Giving logo is copyright `360Giving <https://threesixtygiving.org>` and may not be used without express permission.
```
Comment (BibianaC) [inline LICENCE.rst:6]: Should this line not be part of the `Note`?

Comment (drkane) [reply LICENCE.rst]: Good point - now changed.

**Pull request merged**